src/content/developers/docs/networking-layer/portal-network/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: The Portal Network
+description: An overview of the Portal Network - an in-development network designed to support low-resource clients.
+lang: en
+---
+
+Ethereum runs on nodes. A "node" is any instance of Ethereum client software connected to other computers also running Ethereum software, forming a network. A client is an implementation of Ethereum that verifies data against the protocol rules and keeps the network secure.
+
+Nodes have to run client software, for example, Prysm and Geth, which are vital to the operations of Ethereum.
+
+Ethereum is a decentralized network that relies on a consensus mechanism called Proof-of-Stake (PoS) to validate and agree upon transactions and the state of the blockchain. Clients are implementations of the Ethereum protocol that enable network participants, known as validators, to contribute their computational resources to secure the network and maintain consensus.
+
+## Limitations of the current networking design {#limitations-of-current-network-design}
+
+Node Operators use clients to store a copy of a part or the entire blockchain and validate transactions and smart contracts. By running their nodes, users can independently verify the state of the network without relying on centralized entities.
+
+Clients are currently developed on top of peer-to-peer technology called DevP2P and LibP2P.
+
+The clients do a great job of running Ethereum; they must work within the design framework of devP2P and libP2P.
+
+![devp2p and libp2p](portal-network-devp2p-libp2p.png)
+
+This means the current clients need more support in what they can do to help keep network data requirements low.
```

Comment:
```suggestion
This means running nodes can be expensive, because a large amount of disk space has to be dedicated to Ethereum. There are several solutions to this problem on the Ethereum roadmap, including [history expiry](https://ethereum.org/en/roadmap/statelessness/#history-expiry), [state expiry](https://ethereum.org/en/roadmap/statelessness/#state-expiry) and [statelessness](https://ethereum.org/en/roadmap/statelessness/). However, these are likely several years away from being implemented. There are also [light nodes](https://ethereum.org/en/developers/docs/nodes-and-clients/light-clients/) that do not save their own copy of the chain data, they request the data they need from full nodes. However, this means light nodes have to trust full nodes to provide honest data and also stresses the full nodes that have to serve the data the light nodes need.
```